index.html

```diff
@@ -612,10 +633,12 @@ <h4 class="subtitle is-6">Our way to success</h4>
           </div>
         </div>
       </div>
-      <button class="modal-close is-large"></button>
+      <!-- WARNING: better to use button tag, added for HW -->
+      <div role="button" class="modal-close is-large"></div>
```

Comment:
No need to do this, if it is a button, it should be a button even if you want to try `role` attr. You already did it correctly when you added `role="alert"` for input addition message in the form